contribute/pull-request.md

```diff
@@ -0,0 +1,99 @@
+
+- Learn how to start [Contributing to Grafana](https://grafana.com/docs/contribute/overview/).
+- Make sure your code follows the relevant [style guides](/contribute/style-guides).
+
+## Your first pull request
+
+If this is your first time contributing to a open-source project on GitHub, make sure you read about [Creating a pull request](https://help.github.com/en/articles/creating-a-pull-request).
+
+To increase the chance of having your pull request accepted, make sure your pull request follows the guidelines:
+
+- Title and description must match the implementation.
+- Pull request must follow the [style guide](#style-guide).
+- Commits within the pull request should follow the [style guide](#style-guide).
+- Pull request should close one related issue.
+- Pull request should contain necessary tests that verify the intended behavior.
+- If your pull request have conflicts, rebase your branch onto the master branch.
+
+If the pull request fixes a bug:
+
+- Pull request description must include `Closes #<issue>` or `Fixes #<issue>`.
+- Pull request should include tests that replicate the fixed bug, to avoid regressions.
+
+### Frontend-specific guidelines
+
+- Pull request doesn't increase the Angular code base.
+- Pull request doesn't contain uses of `any` or `{}` without comments describing why.
+- Pull request doesn't contain large React components that could easily be split into several smaller components.
+- Pull request doesn't contain backend calls directly from components—use actions and Redux instead.
+- Pull request uses [Emotion](https://github.com/grafana/grafana/blob/master/style_guides/styling.md) for styling.
+
+#### Redux
+
+- Pull request doesn't contain code that mutates state in reducers or thunks.
+- Pull request uses the `actionCreatorFactory` and `reducerFactory` helpers instead of traditional switch statement reducers in Redux. Refer to [Redux framework](https://github.com/grafana/grafana/tree/master/style_guides/redux.md) for more details.
+- Pull request uses `reducerTester` to test reducers. Refer to [Redux framework](https://github.com/grafana/grafana/tree/master/style_guides/redux.md) for more details.
+- Pull request doesn't contain code that accesses the reducers state slice directly. Instead, the code should use state selectors to access state.
+
+## Code review
+
+Once you've created a pull request, the next step is to have someone review your change. A review is a learning opportunity for both the reviewer and the author of the pull request.
+
+If you think a specific person needs to review your pull request, you can tag them in the description or in a comment. Tag a user by typing the `@` symbol followed by their GitHub username.
+
+We recommend that you read [How to do a code review](https://google.github.io/eng-practices/review/reviewer/) to learn more about some best practices for code reviews.
+
+## Style guide
```

Comment:
This might be confusing, since we have several other things called "style guide." Perhaps "Format Guidelines"? Or just delete the heading and paragraph, then move the subheadings up one level.